ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -30,4 +44,37 @@
 <InfoTableRow @label="Default lease TTL" @value={{@mountConfig.config.defaultLeaseTtl}} />
 <InfoTableRow @label="Max lease TTL" @value={{@mountConfig.config.maxLeaseTtl}} />
 <InfoTableRow @label="Allowed managed keys" @value={{or @mountConfig.config.allowedManagedKeys "None"}} />
-<div class="has-top-margin-l"></div>
\ No newline at end of file
+<div class="has-top-margin-l"></div>
+
+{{#if this.showDeleteAllIssuers}}
+  <Modal
+    @title="Delete All Issuers?"
+    @type="warning"
+    @onClose={{fn (mut this.showDeleteAllIssuers) false}}
+    @isActive={{this.showDeleteAllIssuers}}
+    @showCloseButton={{true}}
+  >
+    <section class="modal-card-custom has-padding-m">
+      This endpoint deletes all issuers and keys within the mount. It is highly recommended to use the individual delete
+      operations instead. This mount will be unusable until new issuers and keys are provisioned.
+      <div class="has-top-margin-m">
+        <h4 class="title is-6 has-bottom-margin-xs">Confirm</h4>
+        <span class="is-hint">Type <b>delete-all</b> to confirm deleting the role.</span>
+        <div class="has-top-margin-xs">
+          <Input class="input" @type="text" @value={{this.confirmDeleteAllIssuersVal}} data-test-delete-all-issuers-input />
+        </div>
+      </div>
+    </section>
+
+    <footer class="has-background-gray-100 has-padding-m">
+      <button
+        type="button"
+        class="button has-text-red"
```

Comment:
I just tried `has-text-danger` - it's slightly different from the designs since it adds a border around the button. I'll use this class since `has-text-red` on hover doesn't produce the correct effect. Thanks for pointing that out!